### DISCUSSION TITLE
How to install manually

### DISCUSSION THREAD

**Discussion by spartanhooah** [starship/starship#2138] [Q&A] [answered]
I'm evaluating starship for use on macOS at work. I was provided with a starship-0.47.0.tar.gz as part of the evaluation process. I extracted the tar.gz and got a folder with several subdirectories (docs, install, media, src, starship_module_config_derive) and some loose files (e.g. Cargo.lock, Cargo.toml). What do I need to do from this point?

**Comment by matchai** [comment, marked as answer]
It sounds like you were given a tar.gz of the repo, rather than our CI build output.
If you'd rather be given a binary, you're free to download them directly from our [release page](https://github.com/starship/starship/releases), along with their associated SHAs.
Otherwise, you can choose to build it yourself locally. You'll need to have Rust available locally, which can be installed with [rustup.rs](https://rustup.rs/). Once in the project root, you can simply run `cargo install --path=. --force` to proceed to build and install it locally.

Let us know if you still need a hand after giving it a shot on your own. 😊

**Reply by spartanhooah** [reply]
Thanks! I'll have the open-source guardian guy grab a release for me instead.
